packages/stream-model/src/model.ts

```diff
@@ -269,36 +229,6 @@ export class Model extends Stream {
     return model
```

Comment:
I think the checks from line 263-268 here can be removed.  `load()` no longer needs to check for incomplete models as it should be impossible to create incomplete models in the first place.

Reply:
good call